modules/ROOT/pages/amazon/amazon-dynamodb-connector-reference.adoc

```diff
@@ -787,68 +785,68 @@ FIXME: Add docs.
 * <<config>> 
 
 ==== Throws
-* DYNAMODB:ACCESS_DENIED_EXCEPTION 
-* DYNAMODB:CONDITIONAL_CHECK_FAILED_EXCEPTION 
-* DYNAMODB:CONNECTIVITY 
-* DYNAMODB:INCOMPLETE_SIGNATURE_EXCEPTION 
+* DYNAMODB:THROTTLING_EXCEPTION 
+* DYNAMODB:UNRECOGNIZED_CLIENT_EXCEPTION 
+* DYNAMODB:RESOURCE_IN_USE_EXCEPTION 
 * DYNAMODB:ITEM_COLLECTION_SIZE_LIMIT_EXCEEDED_EXCEPTION 
-* DYNAMODB:LIMIT_EXCEEDED_EXCEPTION 
-* DYNAMODB:MISSING_AUTHENTICATION_TOKEN_EXCEPTION 
 * DYNAMODB:PROVISIONED_THROUGHPUT_EXCEEDED_EXCEPTION 
-* DYNAMODB:RESOURCE_IN_USE_EXCEPTION 
-* DYNAMODB:RESOURCE_NOT_FOUND_EXCEPTION 
+* DYNAMODB:INCOMPLETE_SIGNATURE_EXCEPTION 
+* DYNAMODB:LIMIT_EXCEEDED_EXCEPTION 
 * DYNAMODB:RETRY_EXHAUSTED 
-* DYNAMODB:THROTTLING_EXCEPTION 
 * DYNAMODB:UNKNOWN 
-* DYNAMODB:UNRECOGNIZED_CLIENT_EXCEPTION 
+* DYNAMODB:RESOURCE_NOT_FOUND_EXCEPTION 
+* DYNAMODB:CONDITIONAL_CHECK_FAILED_EXCEPTION 
 * DYNAMODB:VALIDATION_EXCEPTION 
+* DYNAMODB:ACCESS_DENIED_EXCEPTION 
+* DYNAMODB:MISSING_AUTHENTICATION_TOKEN_EXCEPTION 
+* DYNAMODB:CONNECTIVITY 
 
 
 [[updateItem]]
 === Update Item
 `<dynamodb:update-item>`
 
-
-FIXME: Add docs.
-
++++
+ Edits an existing item's attributes, or adds a new item to the table if it does not already exist.   <a href="https://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_UpdateItem.html">API Reference</a> 
```

Comment:
It's not clear what "it" refers to. Does it refer to an attribute? If so, 

"...or adds a new attribute to the table..."

Reply:
The sentence is correct. This operation edit an existing item's attributes but also can add new item to the table if this item does not already exist.